src/targeting.js

```diff
@@ -64,9 +65,28 @@ targeting.setTargeting = function(targetingConfig) {
   });
 };
 
+/**
+ * normlizes input to a `adUnit.code` array
+ * @param  {(string|string[])} adUnitCode [description]
+ * @return {string[]}     AdUnit code array
+ */
+function getAdUnitCodes(adUnitCode) {
+  if (typeof adUnitCode === 'string') {
+    return [adUnitCode];
+  }
+  else if (utils.isArray(adUnitCode)) {
+    return adUnitCode;
+  }
+  return $$PREBID_GLOBAL$$._adUnitCodes;
```

Comment:
It should not be possible to get to this point and still have `pbjs._adUnitCodes` filter undefined but might want to test for that or otherwise assure the return value here, if undefined, is handled.